main.js

```diff
@@ -4,55 +4,56 @@ var message = document.querySelector('.message')
 var receiveMessageButton = document.querySelector('.receive-message');
```

Comment:
I don't know if this is something they will ask of you, but we were asked to only use ids for query selectors. Unless, using a class name is warranted (e.g., when you want to use querySelectorAll()).